internal/updater/providers/slickvpn/url.go

```diff
@@ -0,0 +1,58 @@
+package slickvpn
+
+import (
+	"strings"
+	"unicode"
+)
+
+func parseOpenvpnURL(url, protocol string) (country, region, city string) {
+	lastSlashIndex := strings.LastIndex(url, "/")
+	url = url[lastSlashIndex+1:]
+
+	suffix := "." + strings.ToUpper(protocol) + ".ovpn"
+	url = strings.TrimSuffix(url, suffix)
+
+	parts := strings.Split(url, ".")
+
+	switch len(parts) {
+	case 1:
+		country = parts[0]
+		return country, "", ""
+	case 2: //nolint:gomnd
```

Comment:
My apologies for `gomnd` being so pushy sometimes

Reply:
No worries, I'm always happy to see good linting in place!